examples/networkpolicy/README.md

```diff
@@ -35,29 +35,25 @@ Once the template has been successfully deployed, following the [simple policy t
 ### Update guidance for clusters deployed by acs-engine releases prior to 0.17.0
 Clusters deployed with calico networkPolicy enabled prior to `0.17.0` had calico `2.6.3` deployed, and a daemonset with an `updateStrategy` of `Ondelete`.
 
-acs-engine releases starting with 0.17.0 now produce an addon manifest for calico in `/etc/kubernetes/addons/calico-daemonset.yaml` contaning calico 3.1.x, and an `updateStrategy` of `RollingUpdate`. Due to breaking changes introduced by calico 3, one must first migrate through calico `2.6.5` or a later 2.6.x release in order to migrate to calico 3.x. as described in the [calico kubernetes upgrade documentation](https://docs.projectcalico.org/v3.0/getting-started/kubernetes/upgrade/).
+acs-engine releases starting with 0.17.0 now produce an addon manifest for calico in `/etc/kubernetes/addons/calico-daemonset.yaml` contaning calico 3.1.x, and an `updateStrategy` of `RollingUpdate`. Due to breaking changes introduced by calico 3, one must first migrate through calico `2.6.5` or a later 2.6.x release in order to migrate to calico 3.1.x. as described in the [calico kubernetes upgrade documentation](https://docs.projectcalico.org/v3.1/getting-started/kubernetes/upgrade/). The acs-engine manifest for calico uses the [kubernetes API datastore, policy-only setup](https://docs.projectcalico.org/v3.1/getting-started/kubernetes/upgrade/upgrade#upgrading-an-installation-that-uses-the-kubernetes-api-datastore).
 
-To get up and running with the new version of calico after upgrading a cluster with acs-engine `0.17.0` and up, follow these steps:
+1. To update to `2.6.5+` in preparation of an upgrade to 3.1.x as specified, edit `/etc/kubernetes/addons/calico-daemonset.yaml` on a master node, replacing `calico/node:v3.1.1` with `calico/node:v2.6.10` and `calico/cni:v3.1.1` with `calico/cni:v2.0.6`. Run `kubectl apply -f /etc/kubernetes/addons/calico-daemonset.yaml`.
```

Comment:
The place where you diverge from the upgrade guidance is here because you are only replacing the image versions, not the entire template/manifest.  The `calico-daemonset.yaml` is a merge of the Calico RBAC + manifest files and needs more changes than just bumping the image versions.

Reply:
It would be easiest if you reference the updated calico-daemonset file already created (per the link above) and then you'd only really need to replace the values which get auto-generated during ACS-Engine runtime for the CIDR.. look for the `<...>` in the template.